dbms/src/DataStreams/HashJoinProbeBlockInputStream.cpp

```diff
@@ -47,21 +49,24 @@ Block HashJoinProbeBlockInputStream::getTotals()
 Block HashJoinProbeBlockInputStream::getHeader() const
 {
     Block res = children.back()->getHeader();
-    join_probe_actions->execute(res);
+    if (res.rows() != 0)
```

Comment:
when will `res.rows() != 0` become true?

Reply:
> when will `res.rows() != 0` become true?

it will be happen in unit tests, I will check why res.rows() != 0.

Reply:
> when will `res.rows() != 0` become true?

It is because that getHeader() return block rows is not 0 in MockTableScanBlockInputStream, I will fix it.

Reply:
fixed.